instrumentation/openfeign-9.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/openfeign/OpenFeignAttributesGetter.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.openfeign;
+
+import feign.Request;
+import feign.Response;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpClientAttributesGetter;
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetClientAttributesGetter;
+import io.opentelemetry.instrumentation.api.instrumenter.rpc.RpcAttributesGetter;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+
+enum OpenFeignAttributesGetter
+    implements
+        RpcAttributesGetter<ExecuteAndDecodeRequest>,
+        NetClientAttributesGetter<ExecuteAndDecodeRequest, feign.Response>,
+        HttpClientAttributesGetter<ExecuteAndDecodeRequest, feign.Response> {
+  INSTANCE;
+
+  @Override
+  public String system(ExecuteAndDecodeRequest request) {
+    return "openfeign";
```

Comment:
I believe this would need to be added to the spec

Reply:
Maybe it should be called "feign"? 
The name of the released version 9.x is no longer found in the official repositories.
[https://github.com/OpenFeign/feign/releases?page=3](url)